<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisibilityToolPropertiesView</class>
 <widget class="QWidget" name="VisibilityToolPropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonSetVisibilityPoint">
     <property name="text">
      <string>Set Visibility Point</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SliderWidget" name="sliderWidgetAreaSize" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSetVisibilityArea">
     <property name="text">
      <string>Set Visibility Area</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSaveTexture">
     <property name="text">
      <string>Save Texture</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>241</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>SliderWidget/SliderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
